docs/dataset/quickstart.md

```diff
@@ -5,3 +5,185 @@ icon: kolena/flame-16
 # :kolena-flame-20: Quickstart
 
 Install Kolena to set up rigorous and repeatable model testing in minutes.
+
+In this quickstart guide, we'll use we will use the [300 Faces In the Wild (300-W)](https://ibug.doc.ic.ac.uk/resources/300-W/)
+example dataset to demonstrate the how to curate test data and test models in Kolena. The examples
+use data located in a public S3 bucket `s3://kolena-public-examples`
+
+## Installing `kolena` Python SDK
+
+=== "`pip`"
+
+    ```shell
+    pip install kolena
+    ```
+
+=== "`poetry`"
+
+    ```shell
+    poetry add kolena
+    ```
+
+## Clone the examples
+
+The [kolenaIO/kolena](https://github.com/kolenaIO/kolena) repository contains a number of example integrations to clone
+and run directly:
+
+<div class="grid cards" markdown>
+
+- [:kolena-keypoint-detection-20: Example: Keypoint Detection ↗](https://github.com/kolenaIO/kolena/tree/trunk/examples/dataset/keypoint_detection)
+
+    ![Example image and five-point facial keypoints array from the 300 Faces in the Wild dataset.](../assets/images/300-W.jpg)
+
+    ---
+
+    End-to-end face recognition 1:1 using the [Labeled Faces in the Wild (LFW)](http://vis-www.cs.umass.edu/lfw/) dataset.
```

Comment:
Missing a blank line here — will render side-by-side as cards when there is a blank line:

![Uploading Screenshot 2024-01-04 at 2.16.37 PM.png…]()

Reply:
Thinking more about this, we should be OK to omit entirely. This has been annoying to keep up-to-date (images are not uniform in appearance, size, or dark mode/light mode support) and doesn't add a ton of value over listing a few examples and pointing to the public repo